Geometry/CommonDetUnit/interface/MuonGeomDet.h

```diff
@@ -0,0 +1,25 @@
+#ifndef CommonDet_MuonGeomDet_H
```

Comment:
ok, thanks. I'd think, this class should be a part of a separate PR for MuonGeomDet migration.